sources/packages/api/src/route-controllers/institution/institution.institutions.controller.ts

```diff
@@ -52,12 +161,358 @@ export class InstitutionInstitutionsController extends BaseController {
   @IsInstitutionAdmin()
   @Patch()
   async update(
-    @Body() payload: InstitutionContactDTO,
+    @Body() payload: InstitutionContactAPIInDTO,
     @UserToken() userToken: IInstitutionUserToken,
   ): Promise<void> {
     await this.institutionService.updateInstitution(
       userToken.authorizations.institutionId,
       payload,
     );
   }
+
+  /**
+   * Create an Institution user.
+   * @param payload
+   * @param user
+   * @returns Primary identifier of the created resource.
+   */
+  @ApiUnprocessableEntityResponse({
+    description:
+      "User to be added either not found in account service or does not belong to same institution of logged in user.",
```

Comment:
Can we mention here _BCeID account service_ instead of only _account service_? It could help someone looking from outside to figure it out easier.

Reply:
I had the same feeling while moving the code 😊. Can change it.